src/Console/Migrate.php

```diff
@@ -59,22 +103,42 @@ protected function migrateTable(string $table, string $column): void
 
     protected function tables(): array
     {
-        return $this->source->getAllTables();
+        return $this->tableNames[0] != '-'
+            ? $this->tableNames
+            : ($this->getTablesFromTarget
+                ? $this->target->getAllTables()
+                : $this->source->getAllTables()
+            );
     }
 
     protected function cleanTargetDatabase(): void
     {
+        if (!$this->dropTables)
+            return;
+
         $this->info('Clearing the target database...');
 
         $this->target->dropAllTables();
     }
 
     protected function runMigrations(): void
     {
+        $runMigrationOn = $this->choice(
+            'Please choose option to run migration on which connection?',
+            ['target', 'both', 'none'],
```

Comment:
1. Move the question to a separate method.
2. Move options to a property.
3. Write the question in one line.